lib/modules/dosomething/dosomething_signup/dosomething_signup.forms.inc

```diff
@@ -98,11 +98,14 @@ function dosomething_signup_form_submit($form, &$form_state) {
   }
 
   if (variable_get('rogue_collection', FALSE)) {
+    $form_state['values']['source'] = 'phoenix-web';
```

Comment:
Oh hmm, are we still saving records in Phoenix when forwarding signups to Rogue? 😲